tests/plugins/autogrow/autogrow.js

```diff
@@ -8,7 +8,58 @@
 
 	bender.editor = {};
 
+	bender.createEditor = function( width, paragraphs ) {
```

Comment:
This method does much more than creating an editor. It set up the whole testing environment and does assertions.